internal/worker/ibcli-exec.go

```diff
@@ -25,6 +25,11 @@ type ImageBuilderArgs struct {
 	//  - repositories
 }
 
+// var wrapper for exec.Command() that can be mocked for testing.
+var execCommand = func(name string, arg ...string) *exec.Cmd {
+	return exec.Command(name, arg...)
+}
```

Comment:
Nitpick: You could simply assign `exec.Command` to `execCommand`, without wrapping it in an additional function, no?